app/routes/users+/$username_+/notes.$noteId.tsx

```diff
@@ -27,22 +30,35 @@ export async function loader({ request, params }: DataFunctionArgs) {
 
 export default function NoteRoute() {
 	const data = useLoaderData<typeof loader>()
+	const date = new Date(data.note.updatedAt)
+	const timeAgo = formatDistanceToNow(date)
```

Comment:
We need to do this computation in the `loader` to avoid timezone issues.

Reply:
Would you mind clarifying? The datetime I'm referencing here _is_ set in the loader and includes the timezone info, so _I think_ it should always show the accurate relative time, respective to the user's current local time.

I could be missing something here though, so please let me know.